### PR TITLE
Storage drawers and cleanup

### PR DESCRIPTION
**Pull request by valipopescu**
added drawers, cleaned up the scripts, added a brass mechanisms recipe in press and compacting

### CODE REVIEW

**Comment by radiomike** [comment]
Think you also need to remove `kubejs\server_scripts\loot_tables.js` as you've replaced it with `z_loot_tables.js`

**Comment by valipopescu** [comment]
> 

if you could tell me which ones are they or if you could remove from the master then i can rebase the repo and reopen the PR ... alternatively can do it after approving since I don't think you'll put this as the only change :).
Removed the loot tables js file

**Comment by valipopescu** [comment]
is this fine ?

File: kubejs/assets/thermal/lang/en_us.json
Comment (radiomike) [inline kubejs/assets/thermal/lang/en_us.json:1]: We need to keep this one

File: kubejs/assets/thermal/en_us.json
[the remark this answers is not in the document]
Comment (radiomike) [reply kubejs/assets/thermal/en_us.json:1]: This is the one to remove

**Review by thevortex — APPROVED** [review]

**Pull request merged**